internal/aws/users.go

```diff
@@ -43,10 +43,11 @@ func NewUser(firstName string, lastName string, email string, active bool) *User
 			FamilyName: lastName,
 			GivenName:  firstName,
 		},
-		DisplayName: strings.Join([]string{firstName, lastName}, " "),
-		Active:      active,
-		Emails:      e,
-		Addresses:   a,
+		DisplayName:   strings.Join([]string{firstName, lastName}, " "),
+		RedisUsername: strings.ReplaceAll(firstName, " ", "") + lastName,
```

Comment:
I have some problems with this solution:
I don't like the idea to modify the codebase on this level. 

This won't work on the go level until you changed the User struct in schema.go!
Have you checked that SCIM supports this modification on the server side? Probably AWS won't allow this kind of SCIM schema change on the client side. 
https://docs.aws.amazon.com/singlesignon/latest/developerguide/limitations.html
Maybe Nickname would work...